tests/kuttl/tests/ctlplane-tls-custom-route/05-check-cert-deletion.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      echo "Checking absence of barbican-public-route certificate..."
+      if oc get certificate barbican-public-route -n $NAMESPACE; then
+        echo "barbican-public-route certificate is found"
+        exit 1
+      else
+        echo "barbican-public-route certificate is absent"
+      fi
```

Comment:
we can do that via the errors, like we check in the cleanup that resources are gone. https://github.com/openstack-k8s-operators/openstack-operator/blob/main/tests/kuttl/tests/ctlplane-tls-custom-issuers/11-errors-cleanup.yaml